README.md

```diff
@@ -2,6 +2,14 @@
 
 Migrate your Ember app to build with Vite.
 
+- [Promise and compatibility](#promise-and-compatibility)
```

Comment:
I don't know if I'm being pedantic here but this TOC looks kind of out of place just dumped here like this 🤔 Maybe literally adding `Table of contents:` before it might make it look less out of place. Do you have any thoughts?